README.md

```diff
@@ -0,0 +1,35 @@
+# Angle Calculator
+
+Investing angle maths (sin, cos, tan etc.) with graphing in matplotlib.
+ 
+## Getting Started
+
+To get started with this project, follow these steps:
+
+1. **Clone the Repository**: 
+
+```bash
+git clone <repository_url>
+
+python -m venv myenv
+
+# run this command if on windows
+myenv\Scripts\activate
+
+# or this if on mac or linux
+source myenv/bin/activate
+```
+
+# To install dependencies
+
+Run `pip install -r requirements.txt` or `pipenv install`
+
+# To test setup
+
```

Comment:
I think once you are up and running you can remove this comment and the file that goes with it.